documentation/STRUCTURE.md

```diff
@@ -0,0 +1,195 @@
+# Documentation
+
+For the purpose of this proposal document we consider all of the following as documentation:
+
+- documentation in our code within the Fairlearn repository which is currently also hosted on readthedocs
+- Jupyter notebooks
+- project website
+- blog posts
+
+Throughout the document we compare with the documentation of other popular projects such as 
+
+- [scikit-learn](https://scikit-learn.org/)
+- [pandas](https://pandas.pydata.org/)
+
+## Goals
+
+The documentation should be
+
+- discoverable, ideally in a single place as opposed to multiple
+- clear
+- concise when describing individual pieces of functionality
+- detailed when describing entire application scenarios, e.g. in the form of case studies
+- available for the latest version, but if possible also for past versions ([example](https://scikit-learn.org/dev/versions.html))
+- maintainable: it should be simple for maintainers to update/validate
+- without ads (readthedocs always has ads that are shown alongside our documentation)
+
+## Proposal
+
+Like for most projects the website, [fairlearn.org](http://fairlearn.org) will be the central place to look for documentation.
+From there visitors have various paths to explore content depending on what they are looking for, as detailed in the following subsections.
+
+### Homepage
+
+```
+website
+|--- Quickstart
+|--- API reference
+|--- User guide
+|--- Example Notebooks
+|--- Contributor guide
+|--- Community
+```
```

Comment:
Some other substance that would be helpful in some place is:

- Who this project serves
- Who is using this project now, what do they value about it
- What other tools or approaches do current users also use
- What's the roadmap

Reply:
@romanlutz added thoughts on "What feels like success" to the team in https://github.com/kevinrobinson/fairlearn/pull/1/files#r420978039

Reply:
@romanlutz also added thoughts on the roadmap and milestones in https://github.com/kevinrobinson/fairlearn/pull/1/files#r420980548.

FWIW, as an interested outsider, one way to evaluate what's happening is a project is to look at open issues and milestone issues in GitHub, and then compare that to what's talked about in chat rooms, and what PRs and commits are actually shipping.  After doing those things myself, I took my best guess at what the project's goals are for the next three months, and for the next year, and I wrote them out here: https://github.com/kevinrobinson/fairlearn/pull/1/files#diff-a594ce9af6d9751647bbd4efefa65312R63.  But it's mostly guessing and I am sure its inaccurate :)

Reply:
@MiroDudik do you have thoughts on this? The more I work on this the more I agree that we need some kind of "About" page that outlines
- vision
- roadmap
- FAQ (like the one @riedgar-ms is creating right now)

That could just be a separate top-level section. I do feel there's some overlap with what we have in "community", which in turn has some overlap with "contributing". Maybe splitting the "community" content into "about" and "contributing" clears this up? Just thinking out loud here, really open to pretty much anything that improves this.

Reply:
As detailed in another comment I've restructured this a little bit, and added an "About" section for this purpose. @riedgar-ms  @MiroDudik  please lmk if you have comments